packages/react/src/hooks/contracts/useContractInfiniteReads.ts

```diff
@@ -1,6 +1,6 @@
 import {
```

Comment:
When I consume this hook, it seems that the page items are of `unknown`.

```tsx
const { data } = useContractInfiniteReads({
  //     ^ `InfiniteData<[unknown]> | undefined`
  cacheKey: 'contracts',
  contracts: (index = 0) => [
    {
      addressOrName: '0x1dfe7ca09e99d10835bf73044a23b73fc20623df',
      contractInterface: mlootContractInterface,
      functionName: 'getChest',
      args: [index],
    },
  ],
  getNextPageParam: (_, pages) => pages.length + 1,
})
```

Reply:
should be fixed now!